.github/workflows/build.yaml

```diff
@@ -83,16 +83,10 @@ jobs:
       - name: Configure Rust Toolchain
         uses: dtolnay/rust-toolchain@stable
```

Comment:
So do we still need rust-toolchain then?

Reply:
Removed, thanks!